core/src/main/java/org/bitcoinj/core/VersionMessage.java

```diff
@@ -113,6 +113,15 @@ public VersionMessage(NetworkParameters params, int newBestHeight) {
         relayTxesBeforeFilter = true;
     }
 
+    /**
+     * Get the service bitfield that represents the node services being provided.
+     *
+     * @return service bitfield
+     */
+    public Services getServices() {
+        return localServices;
+    }
+
```

Comment:
For convenience, consider adding:
```
public boolean has(long feature) {
   return localServices.has(feature);
}   
```

Reply:
I don't understand. That's a recursive loop.

Reply:
Add `VersionMessage.has()` as a shortcut for `VersionMessage.getServices().has()`...